Library/Tasks/AbstractTask.php

```diff
@@ -113,14 +114,16 @@ private function parseOpt(array $paramList)
         $this->lock();
         $this->initWorker();
 
-        $this->kafkaTopicConsumer = $this->kafkaConsumer->newTopic($this->topicName, $this->kafkaTopicConf);
-        $this->kafkaTopicConsumer->consumeStart($this->topicPartitionNo, RD_KAFKA_OFFSET_STORED);
         // xxx Factorize stdout/err support
-        $this->getDI()->get('logger')->info("Worker listening on \033[32m" .
+        $this->getDI()->get('logger')->info(
+            "Worker listening on \033[32m" .
```

Comment:
perhaps we could make a func dedicated to display colored logger's msg

Reply:
right, in another PR 👍